docs/example_static_html.md

```diff
@@ -3,50 +3,190 @@ title: "Example: Plain Old Static HTML"
 layout: docs
 ---
 
-Maybe you want to try Tilt but don't have a suitable project on hand.
+The best indicator of a healthy development workflow is a short feedback loop.
 
-That's OK! Let's look at a project that just serves static HTML.
+Kubernetes is a huge wrench in the works.
 
-These projects can also be useful to confirm Tilt is working as expected in your environment.
+Let's fix this.
 
-## oneup
+In this example, we're going to take you through a very simple shell script that
+serves static HTML.
 
-Oneup is a simple app that uses busybox to serve HTML files.
+We'll use Tilt to:
 
-First, check out the Tilt repo.
+- Run the server on Kubernetes
+- Measure how long it takes to deploy
+- Optimize the updates for fast feedback.
 
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
+as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-html](https://github.com/windmilleng/tilt-example-html)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-html/blob/master/2-recommended/Tiltfile)
+
+## Step 0: The Simplest Deployment
+
+Our server is a two-line shell script:
+
+```shell
+echo "Serving files on port 8000"
+busybox httpd -f -p 8000
 ```
-git clone https://github.com/windmilleng/tilt
-cd tilt/integration/oneup
+
+To start this server on Kubernetes, we need 3 configs:
+
+1) A [Dockerfile](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/Dockerfile) that builds the image.
+
+2) A [Kubernetes deployment](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/kubernetes.yaml) that runs the image
+
+3) And finally, a Tiltfile that ties them together.
+
+```python
+docker_build('example-html-image', '.')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-html', port_forwards=8000)
 ```
 
-We'll be looking at the [oneup](https://github.com/windmilleng/tilt/tree/master/integration/oneup) project
-in `integration/oneup`.
+The first line tells Tilt to build an image with the name `example-html-image`
+in the directory `.` (the current directory).
 
-In the oneup directory, run
+The second line tells Tilt to load the Kubernetes deployment yaml. The image name in the `docker_build`
+call must match the image name in the `example-html` Deployment.
```

Comment:
if you wanna be even more specific, "must match the `spec.container.image` value"

Reply:
mmmm okay yeah fair.